### PR TITLE
Dvanarkel patch to pinenote next

### PR DESCRIPTION
**Pull request by dvanarkel**

### CODE REVIEW

**Comment by iMartyn** [comment]
Hi dvanarkel,  Thanks for the contribution.

File: .github/workflows/base.yml [hunk 1]
```diff
@@ -17,10 +17,10 @@ jobs:
           sudo apt build-dep -y linux-base
           sudo apt install -y gcc-aarch64-linux-gnu binutils-aarch64-linux-gnu git flex bison bc zip jq
       - run: mkdir linux
-      - run: curl -L https://github.com/smaeul/linux/archive/refs/heads/rk356x-ebc-dev.tar.gz | tar -zxf - -C linux --strip-components=1
+      - run: curl -L https://github.com/smaeul/linux/archive/refs/heads/rk35/pinenote-next.tar.gz | tar -zxf - -C linux --strip-components=1
       - run: make -j $(nproc) O=../build/ ARCH=arm64 CROSS_COMPILE=aarch64-linux-gnu- pinenote_defconfig
         working-directory: ./linux
-      - run: make -j $(nproc) O=../build/ ARCH=arm64 CROSS_COMPILE=aarch64-linux-gnu- all
+      - run: make -j 4 O=../build/ ARCH=arm64 CROSS_COMPILE=aarch64-linux-gnu- all
```
Comment (iMartyn) [inline .github/workflows/base.yml]: Why are you proposing changing the parallelism here?

**Comment by dvanarkel** [comment]
Background - I was wanting to trial the pinenote-next kernel to see if that had progressed from the january rk356x-ebc-dev status. Inadvertently created a pull-request in your repo, while the automated build is failing. It fails at a stage I do not understand (segfault compiling empty.c, which is unchanged between the two kernels), so was trialling if it might be an out-of-memory issue related to parallel compiles.  Please discard this PR, I am happy to resubmit if i) I get this working, and ii) the kernel is indeed an improvement. Apologies, Diederik

**Comment by iMartyn** [comment]
Cool, no worries, feel free to reopen if you have success!

**Pull request closed without merging**